### ISSUE TITLE
WMPL compatible

### ISSUE THREAD

**Issue by github52341123** [ThemeFuse/Brizy#86]
I wonder if the Brizy is compatible with WordPress Multilingual Plugin or will this be added? It would be really great. I think it also would be a very good selling point to be added as a Pro feature.

https://wpml.org/documentation/theme-compatibility/go-global-program/

**Comment by michacassola** [comment]
WPML is a mut for me too.

**Comment by michacassola** [comment]
Dear @bagrinsergiu and @DimiBaitanciuc, please join the following program:
https://wpml.org/documentation/theme-compatibility/go-global-program/

I have already contacted WPML support about Brizy compatibility. Please also put in effort to get in touch with them.

**Comment by PriitGit** [comment]
That would be nice!

**Comment by michacassola** [comment]
Please add a Trello card for WPML compatibility.

**Comment by codeclinic** [comment]
So, is this progressing? A Translation &/or WPML specific card is probably a good idea on the Trello board.

Translation integration with the No1 translation plugin is really a must have for something like Brizy.

**Comment by PriitGit** [comment]
Totally agree with that. It must have WPML support.

**Comment by ebenj** [comment]
Waiting for this

**Comment by michacassola** [comment]
I have moved on from Brizy due to the lack of interest of making it WPML compatible. Maybe Brizy will be professionally usable on multilanguage pages in a year from now.

**Comment by DimiBaitanciuc** [comment]
We in contact with the guys from WPML/ The WPML team has someone on their side looking into Brizy as we speak. I'll keep in the loop once I know more.

**Comment by michacassola** [comment]
Hey @DimiBaitanciuc, any news?

**Comment by DimiBaitanciuc** [comment]
The ball is actually in their field so to speak Micha.

I’ve talked with their guys, in fact I was contacted by them, and they are testing it, not sure if it started.

Maybe if you could send them an email, I bet this will move things fw coming from a client and community member :)

Thanks.

All the best,
Dimi

**Comment by codeclinic** [comment]
Ok,,

Thats not great news but I suppose it at least leaves some hope of this coming about.

I've sent a message to WPML, hopefully a few other Brizy / WPML users can do the same.

Thanks.

**Comment by DimiBaitanciuc** [comment]
In the mean time, not sure if you know about our friends at https://translatepress.com/

The founders are personal friends of ours in real life and what they’ve built with Translate Press is really amazing.

I’ll put it this way, what Brizy does for WordPress visual building space, TranslatePress does for visual translation of pages.

In fact I don’t know another translation plugin that does it how they do it.

We are working right now at a closer partnership and I think Translate Press will become our translation plugin of choice shortly.

Give it a try.

(We are yet to fully test it and actually in the process of doing that in the following weeks.)

**Comment by codeclinic** [comment]
Oh great...

So not only this is essentially a big fat advert for your "friends" product, but now also appear to be telling us WPML users that we'Ve wasted hundreds of $$$$ buying the pro version not to mention all the time we've invested in learning the product and the hours spent debugging and working with you to improve Brizy, you now appear to be saying that long term you'll be neglecting all translation software in favour of your "friends" translation product.

**Comment by DimiBaitanciuc** [comment]
Yes, this was the exact point man, you got us ... we created Brizy to get all the cash from the WPML users and I must say it is working wonders. 

Come on, of course we want compatibility with WPML. In fact they have someone looking into it for some time now. I'll follow up again and check the progress. 

As for the the "big fat advert", I don't usually recommend products that I don't love so call it how you like, doesn't change the fact that TranslatePress is a great product. 

Plus I feel like I can recommend it if I want to ;) 

Me knowing the developers personally is a great thing mate because I can vouch for their passion for this product.

No worries no one is forcing you to use a product you don't want to use.

**Comment by michacassola** [comment]
Feel free to promote them more, people have to make money and promoting stuff or yourselve is the only way to do it. 

However I am a friend of lifetime pricing, may it be without support or with a limitation of the number of sites you can isntall it on but I need lifetime updates or I won't use a wordpress software. Unfortunately TranslatePress has a yearly pricing structure that I do not like, maybe you can get us a nice deal AppSumo style @DimiBaitanciuc ?

**Comment by puntorosso** [comment]
I also check on this thread because I don't know if the other one is monitored.
Any update? The folks by WPML told me they are waiting for Brizy's team.
